model-server/src/test/kotlin/org/modelix/model/server/store/IgnitePostgresRepositoryRemovalTest.kt

```diff
@@ -13,33 +13,52 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import org.junit.jupiter.api.AfterAll
+
+package org.modelix.model.server.store
+
 import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.assertDoesNotThrow
 import org.modelix.model.lazy.RepositoryId
-import org.modelix.model.server.store.IgniteStoreClient
-import org.modelix.model.server.store.ObjectInRepository
+import org.testcontainers.containers.PostgreSQLContainer
+import org.testcontainers.utility.DockerImageName
 import java.sql.Connection
 import java.sql.DriverManager
+import java.util.Properties
 import kotlin.test.AfterTest
+import kotlin.test.BeforeTest
 import kotlin.test.assertEquals
 import kotlin.test.assertFalse
 import kotlin.test.assertTrue
 
 class IgnitePostgresRepositoryRemovalTest {
-    private val store = IgniteStoreClient()
+    private lateinit var postgres: PostgreSQLContainer<*>
+    private lateinit var store: IgniteStoreClient
+    private lateinit var dbConnection: Connection
 
     private val toDelete = RepositoryId("repository-removal-toDelete")
     private val existing = RepositoryId("repository-removal-existing")
 
+    @BeforeTest
+    fun beforeTest() {
+        postgres = PostgreSQLContainer(DockerImageName.parse("postgres:16.2-alpine"))
+            .withDatabaseName("modelix")
+            .withUsername("modelix")
+            .withPassword("modelix")
+            .withExposedPorts(5432)
```

Comment:
Chances are high that this port is already used and the test will fail. Tests should use some random non-default port.

Reply:
`withExposedPorts` exposes the specified port from the container to a random free port on the host.

The port on the host is retrieved by calling `postgres.firstMappedPort` in line 51.

Reply:
Nice.